### PR TITLE
Viewer: add analytics

### PR DESCRIPTION
**Pull request by ebidel**
R: all

### CODE REVIEW

**Comment by patrickhulce** [comment]
sweet :)

File: lighthouse-viewer/app/index.html [hunk 1]
```diff
@@ -43,5 +43,14 @@
...
 
 <script src="src/main.js" async></script>
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-85519014-2', 'auto');
+  ga('send', 'pageview');
```
Comment (brendankenny) [inline lighthouse-viewer/app/index.html]: this will capture gist IDs, which we want to avoid

**Comment by paulirish** [comment]
Shall we put in another `ga('send'` for gistCreatedSuccess?  To see the conversion funnel.

File: lighthouse-viewer/app/index.html (hunk 1 above)
Comment (ebidel) [inline lighthouse-viewer/app/index.html]: Filtering them out at the report level. We'll never see them:

screen shot 2016-12-07 at 2 34 44 pm

screen shot 2016-12-07 at 2 37 36 pm

**Comment by ebidel** [comment]
@paulirish done

**Pull request merged**